index.html

```diff
@@ -67,7 +67,7 @@ <h2>RDO Trunk Development</h2>
                   <h3>Lastest Build Reports</h3>
                   <ul>
                     <li>
-                      <a href="https://trunk.rdoproject.org/f25/report.html">Fedora rawhide master</a>
+                      <a href="https://trunk.rdoproject.org/fedora/report.html">Fedora rawhide master</a>
                     </li>
                     <li>
```

Comment:
These old fedora lines refer to a previous builder which used rawhide to build and was removed in favor of fedora28. I think it's good to fix those lines but could you fix also the text, it should be "Fedora 28 Stabilized master" instead of "Fedora rawhide master".